tasks/common/config.yml

```diff
@@ -15,6 +15,7 @@
   with_items: "{{ unit_config }}"
   loop_control:
     loop_var: unit_item
+    label: "{{ unit_item.name }}"
```

Comment:
Nice, worth including the `unit_item.type` within the label as well for better clarification?

e.g. `label: "{{ unit_item.name }}.{{ unit_item.type | default(_default_unit_type) }}"

Reply:
This is an excellent remark! I commit and push this/